src/sonic-yang-models/yang-models/sonic-vxlan.yang

```diff
@@ -0,0 +1,212 @@
+module sonic-vxlan {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-vxlan";
+    prefix svxlan;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+    import sonic-vlan {
+        prefix svlan;
+    }
+	import sonic-types {
+		prefix stypes;
+	}
+
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC VXLAN";
+
+    revision 2021-04-12 {
+        description
+            "First revision.";
+    }
+
+    container sonic-vxlan {
+
+        container VXLAN_TUNNEL {
+
+            description "config db VXLAN_TUNNEL table";
+
+            list VXLAN_TUNNEL_LIST {
+
+                key "name";
+                max-elements 1;
+
+                leaf name {
+                    must "(starts-with(../name, 'vtep'))"
```

Comment:
I don't think this restriction exists today. Please remove this. Might break backward compatibility

Reply:
Sure. Have removed the same.